### PR TITLE
[Site] Main Eder logo in nav should link to homepage

### LINKED ISSUE #65: [Site] Main Eder logo in nav should link to homepage

### PR DESCRIPTION
**Pull request by franzhcs**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #65

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/
- After: https://issue-65--eder-group--hlxsites.hlx.page/

### CODE REVIEW

**Review by mhaack — APPROVED** [review]

**Comment by franzhcs** [comment]
@alexcarol  I exported the `linkPicture` function as the decorated one wasn't working. Not sure if this is intended and I am happy to further debug to make it work on the decorator side - however, I didn't want to break something else in the meanwhile.

Could you please review this?

**Comment by alexcarol** [comment]
> @alexcarol I exported the `linkPicture` function as the decorated one wasn't working. Not sure if this is intended and I am happy to further debug to make it work on the decorator side - however, I didn't want to break something else in the meanwhile.
> 
> Could you please review this?

I did some tests and I see no issues when modifying decorate linked pictures to this:
```
/* used to add links with pictures with word */
export function decorateLinkedPictures(block) {
  block.querySelectorAll('picture').forEach((picture) => {
    linkPicture(picture);
  });
}
```
I initially copied the code from another franklin project, but this looks like it should do the job. What do you think @franzhcs  @mhaack ?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (alexcarol) [inline .gitignore:1]: It's fine to leave, but I'd suggest adding this in a global gitignore file on your machine.

**Review by alexcarol — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (franzhcs) [reply .gitignore:1]: fair enough

**Pull request merged**